large_job_submission.py

```diff
@@ -0,0 +1,77 @@
+"""Submits simulations to the API."""
+import os
+
+import numpy as np
+
+from lib import models
+from lib import scenarios
+
+import inductiva
+
+INPUT_DATASET = "Path/to/meshes"
+WORKING_DIR = None
+
+FLOW_VELOCITY_RANGE_X = [20, 30]
+FLOW_VELOCITY_RANGE_Y = [0, 0]
+FLOW_VELOCITY_RANGE_Z = [0, 0]
+NUM_SIMULATIONS_PER_OBJECT = 1
+
+DOMAIN_X = [-5, 20]
+DOMAIN_Y = [-5, 5]
+DOMAIN_Z = [0, 8]
+NUM_ITERATIONS = 100
+RESOLUTION = 2
+
+MACHINE_TYPE = "c2-standard-16"
+NUM_MACHINES = 1
+DISK_SIZE_GB = 70
+
+
+def run_simulation(object_path, flow_velocity_range_x, flow_velocity_range_y,
+                   flow_velocity_range_z, domain_x, domain_y, domain_z,
+                   num_iterations, resolution, machine_group):
```

Comment:
no need to have all of these arguments. We can use the globals directly.
Remember, this is a script.